packages/core/src/accessibility/focusStyleManager.ts

```diff
@@ -9,7 +9,14 @@ import { InteractionModeEngine } from "../common/interactionMode";
 
 export const FOCUS_DISABLED_CLASS = "pt-focus-disabled";
 
-const focusEngine = new InteractionModeEngine(document.documentElement, FOCUS_DISABLED_CLASS);
+const fakeFocusEngine = {
+    isActive: () => true,
+    start: () => true,
+    stop: () => true,
+};
+const focusEngine = typeof document !== "undefined"
+    ? new InteractionModeEngine(document.documentElement, FOCUS_DISABLED_CLASS)
+    : fakeFocusEngine;
 
 // this is basically meaningless to unit test; it requires manual UI testing
 /* istanbul ignore next */
```

Comment:
I've been using `gulp test`, which is an alias for `gulp karma`. `gulp karma-core` seems to do the same thing but only for the 'core' package. In all cases a coverage report is generated, but it is empty (shows `100% Statements 0/0 100% Branches 0/0 100% Functions 0/0 100% Lines 0/0`). Anyhow, coverage seems to pass now.